### ISSUE TITLE
How to run tests

### ISSUE THREAD

**Issue by marcopus** [mcastorina/mpv-remote-app#63]
Hi @mcastorina, I'm having a hard time running the server tests... Here's the output that I get. Am I missing some configuration?

```
$ make test
python3 -m unittest discover tests
E
======================================================================
ERROR: setUpClass (test_media_server.TestMediaServer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/marcomarra/git/mpv-remote-app/server/tests/test_media_server.py", line 18, in setUpClass
    cls.server.run(daemon=True)
  File "/home/marcomarra/git/mpv-remote-app/server/mpv_remote_app/media_server.py", line 47, in run
    self.open()
  File "/home/marcomarra/git/mpv-remote-app/server/mpv_remote_app/media_server.py", line 42, in open
    self.sock.bind((self.address, self.port))
socket.gaierror: [Errno -2] Name or service not known
```

**Comment by mcastorina** [comment]
It's been awhile since I've looked at the tests 😅

Perhaps the `MediaServer` class got updated without the tests. In fact, it was probably here 77e0c1619cc8088d35dc84182e1c09f86c3973f3 when the `__init__` was changed.

**Comment by mcastorina** [comment]
Fixed in 1b0598e3f540b862cac6f2224075c5cc3622529f

```
» make test
python3 -m unittest discover tests
..................
----------------------------------------------------------------------
Ran 18 tests in 0.197s

OK
```
